src/main/discord_bot.py

```diff
@@ -39,9 +40,13 @@ async def background_tasks(self):
             self.status_check.execute()
         )
 
+__location__ = os.path.realpath(
+    os.path.join(os.getcwd(), os.path.dirname(__file__)))
```

Comment:
the logic for loading a local config file should be in its own module in the util package